.github/CONTRIBUTING.md

```diff
@@ -26,15 +26,32 @@ Generally speaking, you should fork this repository, make changes in your own fo
 ### Working with forks
 
 * [Configure this repository as a remote for your own fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and
+
+```shell
+git remote add upstream git@github.com:sap/jenkins-library.git
+git remote -v
+```
+
 * [Sync your fork with this repository](https://help.github.com/articles/syncing-a-fork/) before beginning to work on a new pull-request.
 
+```shell
+git fetch --all
+git merge upstream/master
+git push
+```
+
 ### Tests
 
 All pipeline library coding _must_ come with automated unit tests.
 
 Besides that, we have an integration test suite, which is not triggered during normal pull request builds. However, integration tests are mandatory before a change can be merged. It is the duty of a team member of the SAP/jenkins-library project to execute these tests.
 To trigger the integration test suite, the `HEAD` commit of the branch associated with the pull request must be pushed under the branch pattern `it/.*` (recommended naming convention: `it/<Number of the pull request>`). As a result, the status `integration-tests` is updated in the pull request.
 
+```shell
+git checkout -B it/<Number of the pull request>
```

Comment:
This here is something which is specific to us, so there can't be any tutorial explaining this!

Anyway this
* can be done by a small group of developers only (with appropriate permissions). For developers "outside" this is pointless.
* is going to be replaced by `/it` comments (ok, currently not working for all of us, reason needs to be investigated)

It is not required to create a local branch `git checkout -B`. Can be simplified to the one-liner `git push upstream <yourPRBranch>:refs/heads/it/<NumberOfPR>`